pom.xml

```diff
@@ -172,6 +172,11 @@
                 <version>${org.powermock.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>com.auth0</groupId>
```

Comment:
use commonly used jwt library across the platform

Reply:
Fixed .